domain-services/ifix-es-pipeline/mvnw.cmd

```diff
@@ -0,0 +1,188 @@
+@REM ----------------------------------------------------------------------------
```

Comment:
Unnecessary file

Reply:
minor change, to be taken up as part of code cleanup.